.clang-format

```diff
@@ -192,4 +192,14 @@ BreakConstructorInitializers: BeforeColon
 # #define bar(y, z)        (y + z)
 #
 AlignConsecutiveMacros: true
+
+
+# The indentation used for requires expression bodies.
+#
+# Example (configuration: Keyword):
+# template <typename T>
+# concept C = requires(T t) {
+#               ...
+#             }
+RequiresExpressionIndentation: Keyword
```

Comment:
Unfortunately, with these changes, the `clang-format` is not executable on F36/F37 where the version of `clang-format` is 14 and 15:

```
.clang-format:204:1: error: unknown key 'RequiresExpressionIndentation'
RequiresExpressionIndentation: Keyword
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Oh I didn't realize that, thanks for catching it.
I have removed the option.